### PR TITLE
kinematic action space converter

### PR DESCRIPTION
**Pull request by tomsilver**
use FK and IK to convert actions in option learning

### CODE REVIEW

File: tests/nsrt_learning/test_option_learning.py [hunk 1]
```diff
@@ -409,3 +412,21 @@ def test_create_action_converter():
...
+    physics_client_id = p.connect(p.DIRECT)
+    robot = create_single_arm_pybullet_robot("panda", physics_client_id)
+    robot.go_home()
+    env_action_arr = np.array(robot.get_joints())
+    converter = create_action_converter()
+    reduced_action_arr = converter.env_to_reduced(env_action_arr)
+    assert reduced_action_arr.shape == (4, )
+    env_action_arr2 = converter.reduced_to_env(reduced_action_arr)
+    reduced_action_arr2 = converter.env_to_reduced(env_action_arr2)
+    assert np.allclose(reduced_action_arr, reduced_action_arr2)
```
Comment (williamshen-nz) [inline tests/nsrt_learning/test_option_learning.py:432]: A check for `np.allclose(env_action_arr, env_action_arr2)` wouldn't hurt

**Review by williamshen-nz — APPROVED** [review]
:shipit:

File: tests/nsrt_learning/test_option_learning.py (hunk 1 above)
Comment (tomsilver) [reply tests/nsrt_learning/test_option_learning.py:432]: isn't it possible that those won't match because FK isn't necessarily invertible?

Comment (williamshen-nz) [reply tests/nsrt_learning/test_option_learning.py:432]: Good point! I forgot about that.

Though I guess in this case, the robot is already at the given position (within the simulator) so IK should give something close by. That being said, we shouldn't check it as that's an implementation detail.

**Pull request merged**